packages/next/src/elements/Nav/index.client.tsx

```diff
@@ -61,7 +61,14 @@ export const DefaultNavClient: React.FC<{
               }
 
               return (
-                <Link className={`${baseClass}__link`} href={href} id={id} key={i} prefetch={false}>
+                <Link
+                  className={`${baseClass}__link ${isActive && 'active'}`}
+                  href={href}
+                  id={id}
+                  key={i}
+                  prefetch={false}
+                >
+                  {isActive && <div className={`${baseClass}__link-indicator`} />}
```

Comment:
You could put the link indicator in the Label component to ensure its always applied consistently.